source/postprocess/particles.cc

```diff
@@ -100,18 +100,12 @@ namespace aspect
             // If the property has dim components, we treat it as vector
             if (n_components == dim)
               {
-#if DEAL_II_VERSION_GTE(9,1,0)
                 vector_datasets.push_back(std::make_tuple(property_index_to_output_index[field_position],
                                                           property_index_to_output_index[field_position]+n_components-1,
                                                           field_name,
                                                           DataComponentInterpretation::component_is_part_of_vector));
-#else
-                vector_datasets.push_back(std::make_tuple(property_index_to_output_index[field_position],
-                                                          property_index_to_output_index[field_position]+n_components-1,
-                                                          field_name));
-#endif
               }
-          }
+	  }
```

Comment:
Conceptually you did everything right now, except that you've removed one space from before the closing brace here. We have a script that indents everything, and only merge patches that have the exact right indentation. Just add the one space back before the brace here and it should work!

Reply:
Hopefully that should do it!